tests/date.test.js

```diff
@@ -0,0 +1,345 @@
+/* eslint-disable max-nested-callbacks */
+/* global describe, it */
+const assert = require("assert");
+
+globalThis.sb = {
+	Date: require("../objects/date")
+};
+
+/**
+ * Generates an Array pre-filled with the index
+ * @param {number} length
+ * @returns {number[]}
+ */
+const simpleRange = (length) => [...new Array(length)].map((_, idx) => idx);
+
+describe("sb.Date", () => {
+	describe("getDaySuffix", () => {
+		it("should return the correct suffix", () => {
+			assert.deepEqual(
+				[1,2,3,4,11,12,13,21,22,23].map(sb.Date.getDaySuffix),
```

Comment:
use `.map(i => sb.Date.getDaySuffix(i))` instead of directly using the function, this is to prevent possible future malfunctions if the API of the method expands